honeybee_radiance/lightsource/sky/hemisphere.py

```diff
@@ -0,0 +1,117 @@
+"""Radiance sky hemisphere."""
+import honeybee_radiance.typing as typing
+import ladybug.futil as futil
+import os
+
+
+class Hemisphere(object):
+    """Radiance sky hemisphere."""
+
+    def __init__(self):
+        """Create sky hemisphere.
+
+        Sky hemisphere is an upward facing glow source. Here is the default sky
+        hemisphere definition. It relies on skyfunc and must be used with one of the
+        Radiance sky commands.
+
+        skyfunc glow sky_glow
+        0
+        0
+        4 1 1 1 0
+        sky_glow source sky
+        0
+        0
+        4 0 0 1 180
+
+        Note:
+        For more information see Chapter `6.3.2  Example: CIE Overcast Sky` in
+        Rendering with Radiance. The chapter is also accessible online at the
+        link below.
+        https://www.radiance-online.org/community/workshops/2003-berkeley/presentations/Mardaljevic/rwr_ch6.pdf
+
+        """
+        self._r_emittance = 1.0
+        self._g_emittance = 1.0
+        self._b_emittance = 1.0
+
+    @property
+    def r_emittance(self):
+        """Sky hemisphere emittance values for red channel (Default: 1.0)."""
+        return self._r_emittance
+
+    @r_emittance.setter
+    def r_emittance(self, value):
+        self._r_emittance = typing.float_in_range(value)
+
+    @property
+    def g_emittance(self):
+        """Sky hemisphere emittance values for green channel (Default: 1.0)."""
+        return self._g_emittance
+
+    @g_emittance.setter
+    def g_emittance(self, value):
+        self._g_emittance = typing.float_in_range(value)
+
+    @property
+    def b_emittance(self):
+        """Sky hemisphere emittance values for blue channel (Default: 1.0)."""
+        return self._b_emittance
+
+    @b_emittance.setter
+    def b_emittance(self, value):
+        self._b_emittance = typing.float_in_range(value)
+
+    @classmethod
+    def from_dict(cls, input_dict):
+        """Create sky_hemisphere from_dict."""
```

Comment:
You might want to put a sample dictionary in the Docstring here.

Reply:
Good idea. Will do.